blogposts/2022/release-3.39

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Sourcegraph 3.39 release'
+publishDate: 2022-04-21T10:00-07:00
+description: 'Sourcegraph 3.39 introduces...'
+tags: [blog, release]
+slug: 'release/3.39'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-39-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-39-release.png
+changelogItems:
+
+---
+
+Sourcegraph 3.39 is now available! In this release:
+
+## Dependencies search now supports GO
+
+Dependencies search beta feature announced in our 3.38 release now supports searching your GO dependencies.
+
+## Code Insights is no longer backed by Timescale
+
+Code Insights can now be used with managed database solutions that do not support [Timescale](https://www.timescale.com/), and it now uses native Postgres. 
+
+This migration has been applied over 3.37 and 3.38, and should be seamless if you are running infrastructure as described by the [official Sourcegraph deployment repositories](https://sourcegraph.com/search?q=context:%40sourcegraph/all+repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph+select:repo&patternType=literal). If you have a customized deployment, you may need to incorporate some changes into your configuration ([more details and examples here](https://github.com/sourcegraph/sourcegraph/issues/32271#issuecomment-1086328666)).
+
+Any managed database solution that supports the [official Sourcegraph Postgres version requirements](https://docs.sourcegraph.com/admin/postgres#version-requirements) is now suitable for Code Insights. Migration from a self-hosted database is not performed automatically, and will need to be performed by a Sourcegraph admin at your own discretion.
```

Comment:
@iskyOS let me know if you think we should include more or less information here – not sure how much we want to call out but I would guess an admin's first thought is "what does this mean for my deployment" and it felt better to at least indicate the answers here directly.

Reply:
@Joelkw Agreed with being direct here, and I think this is a pretty good balance of letting people know what action is required without going too deep.

The big thing this unblocks is using CI with RDS, right? The one thing I'd say we can do here is livening it up to point out why it's exciting, so it wouldn't hurt to directly call out RDS support (or whatever other DBs are common that we're now supporting)

Reply:
Got it, will add that callout! Correct, that's the big one.